esx_service/vsan_policy_test.py

```diff
@@ -61,7 +61,11 @@ def test_create_delete(self):
         self.assertFalse(os.path.isfile(self.policy_path))
 
     def test_delete_nonexistent_policy_fails(self):
-        self.assertNotEqual(None, vsan_policy.delete(self.name))
+	log = logging.getLogger('Delete_nonexistent_policy')
```

Comment:
Can you make this into a utility function for tests to log a message. As we add more tests we will might have to log other messages.